Tests/InfomaniakCoreTests/UTCollectionTests.swift

```diff
@@ -94,6 +94,41 @@ final class UTSendableArray: XCTestCase {
         // THEN
         XCTAssertEqual(collection.count, 2)
     }
+
+    func testRemoveAll() async {
+        // GIVEN
+        let collection = SendableArray<String>()
+        collection.append("a")
+        collection.append("b")
+
+        // WHEN
+        let t = Task.detached {
+            collection.removeAll()
+        }
+
+        _ = await t.result
+
+        // THEN
+        XCTAssertEqual(collection.count, 0)
```

Comment:
```suggestion
        XCTAssertTrue(collection.isEmpty)
```

Reply:
Using more specific `XCTAssertX` helps understanding the test the day it fails and you have to understand what broke.